WORKING_GROUPS.md

```diff
@@ -0,0 +1,249 @@
+participate in a non-voting capacity.
+
+The moderator is responsible for summarizing the discussion of each
+agenda item and sends it as a pull request after the meeting.
+
+### Consensus Seeking Process
+
+The WG follows a [Consensus Seeking][] decision-making model.
+
+When an agenda item has appeared to reach a consensus the moderator
+will ask "Does anyone object?" as a final call for dissent from the
+consensus.
+
+If an agenda item cannot reach a consensus a WG member can call for
+either a closing vote or a vote to table the issue to the next
+meeting. The call for a vote must be seconded by a majority of the WG
+or else the discussion will continue. Simple majority wins.
+
+Note that changes to WG membership require unanimous consensus.  See
+"WG Membership" above.
+
+<a id="developers-certificate-of-origin"></a>
+## Developer's Certificate of Origin 1.1
+
+*Note*: The DCO is mandatory for all Node.js Foundation projects.
+
+By making a contribution to this project, I certify that:
+
+* (a) The contribution was created in whole or in part by me and I
+  have the right to submit it under the open source license
+  indicated in the file; or
+
+* (b) The contribution is based upon previous work that, to the best
+  of my knowledge, is covered under an appropriate open source
+  license and I have the right under that license to submit that
+  work with modifications, whether created in whole or in part
+  by me, under the same open source license (unless I am
+  permitted to submit under a different license), as indicated
+  in the file; or
+
+* (c) The contribution was provided directly to me by some other
+  person who certified (a), (b) or (c) and I have not modified
+  it.
+
+* (d) I understand and agree that this project and the contribution
+  are public and that a record of the contribution (including all
+  personal information I submit with it, including my sign-off) is
+  maintained indefinitely and may be redistributed consistent with
+  this project or the open source license(s) involved.
+
+### Moderation Policy
+
+The [Node.js Moderation Policy]() applies to this WG.
+
+### Code of Conduct
+
+The [Node.js Code of Conduct]() applies to this WG.
+
+[Node.js Code of Conduct]: https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md
+[Node.js Moderation Policy]: https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md
+```
+
+[Community Committee]: https://github.com/nodejs/community-committee
+
+## Current Working Groups
+
+* [Evangelism](#evangelism)
+
+### [Evangelism](https://github.com/nodejs/evangelism)
+
+The Evangelism WG creates promotional materials, brings discussions to a wider audience, and attemps to grow the community through positive, supportive messaging.
+
+Responsibilities include:
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
```

Comment:
Looks like the list of responsibilities for the Evangelism group was copied from the TSC or something. Definitely needs to be updated.

Reply:
This actually comes from the Evangelism WG Governance doc: https://github.com/nodejs/evangelism/blob/master/GOVERNANCE.md

Reply:
This actually comes from the Evangelism WG Governance doc: https://github.com/nodejs/evangelism/blob/master/GOVERNANCE.md

Reply:
@bnb As Champion of the Evangelism WG, please advise. It looks like the GOVERNANCE.md document has some cruft left over.

Reply:
The context in which this is presented is confusing  - there is context in the governance file that these are the responsibilities that the Evangelism WG holds over _the nodejs/evangelism repo_, where "project" is an agnostic name for the contents of the nodejs/evangelism repo.

Reply:
@bnb I went back to the document to try to improve the context. Upon further review, this is simply the boilerplate text from the Node.js (neé io.js) WORKING_GROUPS.md document: https://github.com/dshaw/community-committee/blob/448ad8b7439851a11281f55de6d28752e76a6c5a/WORKING_GROUPS.md#bootstrap-governance

I think it would better to remove the section of detailed responsibilities of the Evangelism WG for now.

That said, I think you should consider iterating on this text with the Evangelism WG. It has been the primary reference we've had to work from in setting up @nodejs/user-feedback.